src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -617,6 +620,7 @@ impl RegionMigrationProcedure {
             from_peer: persistent_ctx.from_peer.clone(),
             to_peer: persistent_ctx.to_peer.clone(),
             timeout: persistent_ctx.timeout,
+            trigger_reason: persistent_ctx.trigger_reason.clone(),
```

Comment:
We can impl `Copy` trait form the `RegionMigrationTriggerReason`

```suggestion
            trigger_reason: persistent_ctx.trigger_reason,
```